### ISSUE TITLE
User cannot change default Post category

### ISSUE THREAD

**Issue by masperber** [Automattic/wp-calypso#58081]
### Quick summary

Initial report: 19236885-hc

When user tries to update their default Post category, no change is made.

### Steps to reproduce

In Calypso, navigate to Settings > Writing in Classic view.
Change default Post category setting and Save Changes.
The change does not save.

Or, navigate to Posts > Categories in WordPress.com vew.
Click on the 3 dots next to a category and attempt to set it as default.
The change does not save.

### What you expected to happen

Expected to be able to change the default category.

### What actually happened

The default category cannot be changed.

### Context

Initial report: 19236885-hc

When user tries to update their default Post category, no change is made.

Attempted from Settings > Writing in Classic view and from Posts > Categories in WordPress.com view.

Browser shows an ERR_BLOCKED_BY_CLIENT message in the console. This error message is not present on a test site. Error message is present on the user's site in both Firefox and Chrome.

### Operating System

Windows, macOS

### Browser

Google Chrome/Chromium, Mozilla Firefox

### Simple, Atomic or both?

Atomic

### Other notes

ERR_BLOCKED_BY_CLIENT message noted in the browser console. This is present on both Firefox and Chrome when viewing the user's site, but not present on an unaffected test site.

### Reproducibility

Once

### Severity

One

### Workaround details

No workaround at present.

**Comment by Robertght** [comment]
Thank you for reporting this @masperber !

I have a few questions to help us narrow this down:

- I can see this is an Atomic site. Were plugins turned off in the process?
- Also, has the wp-admin dashboard been considered?
- Were you able to replicate locally?

**Comment by masperber** [comment]
I just checked the user's site again, and the issue appears to have resolved without user report. There has been a lot of plugin activity since Monday, so it's hard to say what could have been the resolution.

>Were plugins turned off in the process?

Not when I was with the user.

>Also, has the wp-admin dashboard been considered?

Yes, I mentioned in the report that we tried using classic view (wp-admin/options-writing.php).

>Were you able to replicate locally?

I didn't attempt to replicate it on a test site.
